android/feature-now-playing/src/main/java/io/projectnewm/feature/now/playing/NowPlayingScreen.kt

```diff
@@ -70,6 +91,12 @@ fun NowPlayingScreen() {
         Spacer(modifier = Modifier.height(24.dp))
 
         TipIcon()
+
+        if (firstLoad.value) {
+            Player.play(context)
+            firstLoad.value = false
+            isPlaying.value = true
+        }
```

Comment:
Could `LaunchedEffect(Unit) { Player.play(context) }` potentially replace the `firstLoad` state?

Reply:
Updated, I think that works